src/graph/FetchVerticesExecutor.cpp

```diff
@@ -118,9 +118,6 @@ Status FetchVerticesExecutor::prepareTags() {
             if (!tagIdStatus.ok()) {
                 return tagIdStatus.status();
             }
```

Comment:
Seems we don't need it anymore.

Reply:
when `fetch prop on * vid yield tag.prop` need to check the yield

Reply:
I mean the tag id is no more used here.

Reply:
Good catch!